### PR TITLE
Textual change for Tracking code generator page

### PR DESCRIPTION
**Pull request by AltamashShaikh**
### Description:

Textual change for Tracking code generator page.
Fixes: #PG-2730

### Review

* [ ] [Functional review done](https://developer.matomo.org/guides/pull-request-reviews#functional-review-done)
* [ ] [Potential edge cases thought about](https://developer.matomo.org/guides/pull-request-reviews#potential-edge-cases-thought-about) (behavior of the code with strange input, with strange internal state or possible interactions with other Matomo subsystems)
* [ ] [Usability review done](https://developer.matomo.org/guides/pull-request-reviews#usability-review-done) (is anything maybe unclear or think about anything that would cause people to reach out to support)
* [ ] [Security review done](https://developer.matomo.org/guides/security-in-piwik#checklist)
* [ ] [Wording review done](https://developer.matomo.org/guides/pull-request-reviews#translations-wording-review-done)
* [ ] [Code review done](https://developer.matomo.org/guides/pull-request-reviews#code-review-done)
* [ ] [Tests were added if useful/possible](https://developer.matomo.org/guides/pull-request-reviews#tests-were-added-if-usefulpossible)
* [ ] [Reviewed for breaking changes](https://developer.matomo.org/guides/pull-request-reviews#reviewed-for-breaking-changes)
* [ ] [Developer changelog updated if needed](https://developer.matomo.org/guides/pull-request-reviews#developer-changelog-updated-if-needed)
* [ ] [Documentation added if needed](https://developer.matomo.org/guides/pull-request-reviews#documentation-added-if-needed)
* [ ] Existing documentation updated if needed

### CODE REVIEW

File: vue/src/Tag/TagDateInput.vue [hunk 1]
```diff
@@ -166,7 +166,7 @@ export default defineComponent({
       .timepicker({ timeFormat: 'H:i:s' })
       // timepicker triggers a jquery event, not a addEventListener event, so vue doesn't catch
       // it
-      .on('change', (event) => {
+      .on('change', (event: ChangeEvent) => {
```
Comment (AltamashShaikh) [inline vue/src/Tag/TagDateInput.vue:169]: `./console vue:build TagManager` was failing and had to update as below to make it work

File: TagManager.php [hunk 2]
```diff
@@ -609,6 +609,7 @@ public function getClientSideTranslationKeys(&$result)
         $result[] = 'TagManager_GettingStartedContributeTags';
         $result[] = 'TagManager_CreateNewVersionNow';
         $result[] = 'TagManager_TagManager';
+        $result[] = 'TagManager_TagManagerNew';
```
Comment (michalkleiner) [inline TagManager.php]: How about
```suggestion
        $result[] = 'TagManager_MatomoTagManager';
```
?
There might be use for both versions possibly and 'New' suffix doesn't indicate well what it is. Just a suggestion though.

Comment (AltamashShaikh) [reply TagManager.php]: Done

**Review by snake14 — APPROVED** [review]
Code looks good and I was able to view the title change in the tracking code view.

**Pull request merged**